libnavigation-ui/build.gradle

```diff
@@ -92,6 +101,7 @@ dependencies {
   testImplementation dependenciesList.mockk
   testImplementation dependenciesList.robolectric
   testImplementation dependenciesList.json
+  compile project(path: ':libnavigation-core')
```

Comment:
should we use `api` or 'implementation' instead of `compile` here?

Reply:
was added by mistake, removed